ios/FluentUI/Divider/DividerTokens.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import UIKit
+
+// MARK: - DividerStyle
+
+/// Pre-defined styles of the divider
+@objc public enum MSFDividerStyle: Int {
+    case `default`
+    case shadow
+}
+
+/// Pre-defined spacings of the divider
+@objc public enum MSFDividerSpacing: Int {
+    case none
+    case medium
+}
+
+/// Representation of design tokens to dividers at runtime which interfaces with the Design Token System auto-generated code
+/// Updating these properties causes the SwiftUI divider to updates its view automatically
+class MSFDividerTokens: MSFTokensBase, ObservableObject {
+    @Published public var color: UIColor!
+    @Published public var padding: CGFloat!
+
+    var style: MSFDividerStyle
```

Comment:
If we're allowing these to be changed (var instead of let), we should call updateForCurrentTheme() on the didSet closure so that the SwiftUI View will be refreshed when the properties are updated:

For example:
https://github.com/microsoft/fluentui-apple/blob/89daa063880b3622f8f0ea229940e03ea354a24d/ios/FluentUI/Avatar/AvatarTokens.swift#L68

```

    var style: MSFAvatarStyle {
        didSet {
            if oldValue != style {
                updateForCurrentTheme()
            }
        }
    }

    var size: MSFAvatarSize {
        didSet {
            if oldValue != size {
                updateForCurrentTheme()
            }
        }
    }
```